dask_geopandas/sjoin.py

```diff
@@ -46,8 +56,8 @@ def sjoin(left, right, how="inner", op="intersects"):
     if isinstance(right, geopandas.GeoDataFrame):
         right = from_geopandas(right, npartitions=1)
 
-    name = "sjoin-" + tokenize(left, right, how, op)
-    meta = geopandas.sjoin(left._meta, right._meta, how=how, op=op)
+    name = "sjoin-" + tokenize(left, right, how, predicate)
+    meta = geopandas.sjoin(left._meta, right._meta, how=how, predicate=predicate)
```

Comment:
This essentially makes us require geopandas>=0.10 I think? (which is fine for me, but maybe add the lower bounds in setup.py)

Reply:
Ah, true. I'll change the requirements. I think that there's no harm having versions of geopandas and dask-geopandas closely coupled, i.e. we can always require latest geopandas with the latest dask-geopandas.

Reply:
Yes, and especially now for releasing the first non-alpha release, it's fine to require the latest geopandas release.